src/main/java/org/broadinstitute/hellbender/Main.java

```diff
@@ -147,6 +148,7 @@ public Object instanceMain(final String[] args) {
      * Note: this is the only method that is allowed to call System.exit (because gatk tools may be run from test harness etc)
      */
     protected final void mainEntry(final String[] args) {
+        BucketUtils.setGlobalNIODefaultOptions();
```

Comment:
Can you move to `CommandLineProgram.instanceMainPostParseArgs()`, just below the call to `BlockGunzipper.setDefaultInflaterFactory()`? Should also add logger output to `CommandLineProgram.printSettings()` to output the current default retries/reopens for GCS.